tests/tests/test_demo_artifact.py

```diff
@@ -102,6 +103,7 @@ def run_demo_script_up(exit_cond=exit_cond):
 
     # Give the test a timeframe, as the script might run forever,
     # if something goes awry, or the script is not brought down properly.
+    @flaky(max_runs=3)
```

Comment:
We should always have a JIRA ticket associated with any test marked as `flaky`. A comment should be enough.

Reply:
Great idea!